.pre-commit-config.yaml

```diff
@@ -73,7 +73,8 @@ repos:
                 setup[.]cfg$
           exclude: |
             (?x)
-                cpp/src/tsne/cannylab/bh[.]cu$
+                cpp/src/tsne/cannylab/bh[.]cu$|
+                python/cuml/_thirdparty
```

Comment:
Looks like we need one more `cuml` directory here

```suggestion
                python/cuml/cuml/_thirdparty
```

Reply:
A thanks, not sure if the code was moved, or I did a bad rebase :).

Reply:
No worries. Me neither. Just saw the CI error and tried to resolve it :)